docs/index.html

```diff
@@ -1,8 +1,9 @@
 <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>#TwitterThrowback</title>
     <meta name="description" content="See what you've Tweeted on this day in the past.">
+    <link href="https://fonts.googleapis.com/css?family=Maven+Pro:400,500" rel="stylesheet">
```

Comment:
Cool one on importing style this way as the other way blocks rendering

Reply:
Can you briefly explain the differences between the two methods and how it blocks rendering?
I'd like to know, please.

Reply:
Using `@import` will wait till the imported content is fetched. You can read more on that [here](https://stackoverflow.com/a/12380004/5040413)

Reply:
Well, in this case, it doesn't have much effect. It's just preferable to using link

Reply:
> Using `@import` will wait till the imported content is fetched. You can read more on that [here](https://stackoverflow.com/a/12380004/5040413)

But isn't that a good reason to use @import? Because normally you don't want to see the content until the font has loaded

Reply:
Thanks for the info and link :+1: